Makefile

```diff
@@ -397,11 +398,15 @@ deploy-test: _verify_cluster generate-keys
 
 # execute on the host, without skipper
 deploy-service-for-subsystem-test: create-hub-cluster load-image
-	skipper $(MAKE) deploy-test TARGET=kind SERVICE=$(SUBSYSTEM_LOCAL_SERVICE) IP=$(shell ip route get 1 | sed 's/^.*src \([^ ]*\).*$$/\1/;q')
+	skipper $(MAKE) deploy-test SERVICE=$(LOCAL_ASSISTED_SERVICE_IMAGE) IP=$(shell ip route get 1 | sed 's/^.*src \([^ ]*\).*$$/\1/;q')
 	@if [ $(ENABLE_KUBE_API) == "true" ]; then \
 		echo "Deploying assisted-service for subsystem tests in kube-api mode..."; \
-		TARGET=kind skipper make enable-kube-api-for-subsystem; \
+		skipper make enable-kube-api-for-subsystem; \
 	fi
+ifdef DEBUG_SERVICE
+	$(KUBECTL) patch deployment assisted-service --type json -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/livenessProbe"}]' || true
```

Comment:
becuse you using this patch template multiple times 
Either create a function or move it to a json file

Reply:
Good point thanks